lib/cocoapods/installer/analyzer.rb

```diff
@@ -289,7 +289,7 @@ def embedded_target_pod_targets_by_host(aggregate_target, embedded_aggregate_tar
             !aggregate_user_target_uuids.intersection(host_target_uuids).empty?
           end
           embedded_aggregate_target.user_build_configurations.keys.each do |configuration_name|
-            embedded_pod_targets = embedded_aggregate_target.pod_targets.select do |pod_target|
+            embedded_pod_targets = embedded_aggregate_target.pod_targets_for_build_configuration(configuration_name).select do |pod_target|
```

Comment:
hah, nice!

Reply:
I'm assuming this also handles custom build configurations as well?

Reply:
it should! I havent tried it though